src/broker/content-cache.c

```diff
@@ -505,7 +505,7 @@ static void cache_store_continuation (flux_future_t *f, void *arg)
             flux_log (cache->h, LOG_DEBUG, "content store: %s",
                       "backing store service unavailable");
         else
-            flux_log_error (cache->h, "content store");
+            flux_log (cache->h, LOG_CRIT, "content store");
```

Comment:
Should be
```c
flux_log (cache->h, LOG_CRIT, "content store: %s", strerror (errno));
```

Reply:
oh yeah, no strerror output when you take out the error ... my bad, sloppy :P